Radio Garden/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "Tepiloxtl",
+    "id": "085733181730332672"
+  },
+  "url": "radio.garden",
+  "description": {
+    "en": "By bringing distant voices close, radio connects people and places. Radio Garden allows listeners to tune into live radio across entire globe."
+  },
+  "service": "Radio Garden",
+  "version": "1.0",
+  "logo": "https://vikavo.lt/premidpresence/radiogardenglobe.webp",
```

Comment:
You own vikavo or have permission to make changes right?

Reply:
I own vikavo.lt. I can reupload to imgur though per @Bas950 request.

Reply:
It's fine.